components/portal/main/controllers.js

```diff
@@ -182,6 +183,7 @@ define(['angular', 'require'], function(angular, require) {
       vm.hideMainMenu = false;
       vm.footerLinks = FOOTER_URLS;
       vm.openMenuByDefault = false;
+      vm.hideMessagesFeatures = false;
```

Comment:
While parallel with `vm.hideMainMenu`, naming the variable "hide" has a double-negative effect. "hide=false" means the feature is turned on. It might be better to use a variable that states the feature positively, as in "vm.messagesFeature" being true when the feature is turned on and false when it is off/hidden.

Notice this play out in the templates, with `&& !vm.hideMessagesFeatures` for "and if the feature is on".